Sofa/framework/Type/src/sofa/type/Mat.h

```diff
@@ -35,7 +35,7 @@ namespace // anonymous
     constexpr real rabs(const real r)
     {
         if constexpr (std::is_signed<real>())
-            return std::abs(r);
+            return r < 0 ? -r : r;
```

Comment:
The problem is that it is less efficient due to the introduction of a condition. I believe `std::abs` is optimized and avoids branches (bitmask).